src/sentry/static/sentry/app/views/organizationDiscover/result/utils.jsx

```diff
@@ -76,6 +76,24 @@ export function getChartDataByDay(rawData, query) {
   });
 }
 
+/**
+ * Returns the page ranges of paginated tables, i.e. Results 1-100
+ * @param {Object} baseQuery data
+ * @returns {String}
+ */
+export function getResultsPageRange(baseQuery) {
+  const dataLength = baseQuery.data.data.length;
+
+  if (!dataLength) {
+    return '0 results';
+  } else if (baseQuery.query.aggregations.length) {
+    return `${dataLength} ${dataLength === 1 ? 'row' : 'rows'}`;
```

Comment:
Should this be the first condition?

Reply:
👍 , switched to 'rows' instead of 'results' so the terminology is consistent instead